src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -114,7 +114,7 @@ def load_arguments(self, _):
         c.argument('enable_private_cluster', action='store_true')
         c.argument('private_dns_zone')
         c.argument('fqdn_subdomain')
-        c.argument('enable_public_fqdn', action='store_true', is_preview=True)
+        c.argument('disable_public_fqdn', action='store_true')
```

Comment:
Has the name of this parameter `enable_public_fqdn` been directly modified? Will this cause breaking change to users?

Reply:
we change this name because during preview stage, the default behavior is disable_public_fqdn, so we add this `enable_public_fqdn` parameter for create operation.
And after GA, we changed the default behavior of this feature to enabled state. So we provide `disable_public_fqdn` parameter now.

Reply:
I think this breaking change is acceptable since this is aks cli extension and it's preview feature.

Reply:
OK~